recipe/meta.yaml

```diff
@@ -118,15 +120,23 @@ outputs:
   - name: spyder-base
   - name: spyder
     build:
-      noarch: python
+      noarch: python  # [unix]
+      string: "osx_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"    # [osx]
+      string: "linux_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [linux]
```

Comment:
```suggestion
      string: "unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [unix]
```

Reply:
why is this change needed at all?

Reply:
i think all you need is the migraton, all these extra lines, are, extra, unless i'm missing osmething.